app/lib/features/chat_ng/widgets/events/message_event_item.dart

```diff
@@ -36,17 +38,25 @@ class MessageEventItem extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    return Column(
-      crossAxisAlignment:
-          isMe ? CrossAxisAlignment.end : CrossAxisAlignment.start,
-      mainAxisSize: MainAxisSize.min,
-      children: [
-        _buildMessageUI(context, ref, roomId, messageId, item, isMe),
-        _buildReactionsList(roomId, messageId, item, isMe),
-      ],
+    return SwipeTo(
+      onRightSwipe: !isMe ? (_) => _handleReplySwipe(ref, item) : null,
+      onLeftSwipe: isMe ? (_) => _handleReplySwipe(ref, item) : null,
```

Comment:
why have different directional swipe for the same action depending on whose message that is? this feels odd. I'd expect that whatever the message, if I swipe one direction I get one action and the other direction, I get another action. In that case, edit only works for my messages ...
```suggestion
      onRightSwipe: (_) => _handleReplySwipe(ref, item)
      // onLeftSwipe: isMe ? (_) => _handleEditSwipe(ref, item) : null,
```

Reply:
Oh yes, thanks for correction. I confirmed the same UI behaviour on whatsapp, signal use-cases.